doc/api/vm.md

```diff
@@ -1061,7 +1061,8 @@ current V8 isolate, or the main context.
     right away to measure the memory.
     **Default:** `'default'`
 * Returns: {Promise} If the memory is successfully measured the promise will
```

Comment:
While we're editing nearby text, perhaps we can add this missing comma too:

```suggestion
* Returns: {Promise} If the memory is successfully measured, the promise will
```

Reply:
Updated thanks for the review.